web-ui/src/components/guides/GuidesPanel.test.jsx

```diff
@@ -1,11 +1,45 @@
 import React from 'react';
-import { MemoryRouter } from 'react-router-dom';
+import {MemoryRouter} from 'react-router-dom';
 import GuidesPanel from './GuidesPanel';
+import {AppContextProvider} from "../../context/AppContext.jsx";
+
+const initialState = {
```

Comment:
Need to run Prettier on this file.

Reply:
Done in https://github.com/objectcomputing/check-ins/pull/2522/commits/53376b653c97952b74d7b654935e83feb9be539a